docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -359,7 +361,18 @@ _References:_
 
 ## max-worker-connections
 
-Sets the maximum number of simultaneous connections that can be opened by each [worker process](http://nginx.org/en/docs/ngx_core_module.html#worker_connections)
+Sets the [maximum number of simultaneous connections](http://nginx.org/en/docs/ngx_core_module.html#worker_connections) that can be opened by each worker process.
+0 will use the value of [max-worker-open-files](#max-worker-open-files).
+_**default:**_ 16384
```

Comment:
This is the value we currently have. I still think it should be good to enforce this value explicitly in our configmap.
Will be done in https://github.com/Shopify/k8s-cluster-services/pull/975/files#diff-05158f8a8c4078788fa13bdd09ae0a77R151